web/server/entrypoint.go

```diff
@@ -106,7 +106,9 @@ func RunServer(ctx context.Context, args []string, _ logging.Logger) (err error)
 	}
 
 	if argsParsed.Logging {
-		utils.UncheckedError(vlogging.GLoggerCamComp.Start(ctx))
+		if err := vlogging.GLoggerCamComp.Start(ctx); err != nil && argsParsed.Debug {
```

Comment:
[q] Do we need to explicitly check `argsParsed.Debug`? It would be nice if we could just `logger.Debug` and the filtering to stdout would happen by the logging library.

Reply:
We do not! Good call. Changed.